package.json

```diff
@@ -80,6 +82,9 @@
     "webpack": "3.12.0"
   },
   "jest": {
+    "globals": {
+      "__BROWSER_BUILD__": true
```

Comment:
I'm not sure this is required anymore.

Reply:
I should've removed this from this branch, but I did it in the part-2. I'll just leave this as it is. It will be gone when merging part-2 back here.